--- src/db/full_schema.rs	2025-05-02 10:43:24.908278166 +0200
+++ src/db/schema.rs	2025-05-02 10:44:19.764214721 +0200
@@ -1,22 +1,17 @@
 // @generated automatically by Diesel CLI.
 
 pub mod sql_types {
     #[derive(diesel::query_builder::QueryId, Clone, diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
-
-    #[derive(diesel::query_builder::QueryId, Clone, diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "user_role"))]
     pub struct UserRole;
 }
 
 diesel::table! {
     use diesel::sql_types::*;
     use postgis_diesel::sql_types::*;
-    use super::sql_types::Geometry;
 
     places (id) {
         id -> Int4,
         name -> Varchar,
         geometry -> Geometry,
     }
