--- src/db/full_schema.rs	2025-03-12 14:54:20.449542884 +0100
+++ src/db/schema.rs	2025-03-12 14:53:27.686636817 +0100
@@ -1,34 +1,28 @@
 // @generated automatically by Diesel CLI.
 
 pub mod sql_types {
     #[derive(diesel::query_builder::QueryId, Clone, diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
-
-    #[derive(diesel::query_builder::QueryId, Clone, diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "user_role"))]
     pub struct UserRole;
 }
 
 diesel::table! {
     use diesel::sql_types::*;
     use postgis_diesel::sql_types::*;
-    use super::sql_types::Geometry;
 
     places (id) {
         id -> Int4,
         name -> Varchar,
         geometry -> Nullable<Geometry>,
     }
 }
 
 diesel::table! {
     use diesel::sql_types::*;
     use postgis_diesel::sql_types::*;
-    use super::sql_types::Geometry;
 
     posts (id) {
         id -> Int4,
         title -> Varchar,
         body -> Text,
         published -> Bool,
